Legal-Forms/Funerary Procedure and Last Will.txt

```diff
@@ -0,0 +1,33 @@
+[color=#9fed58]╔══════════════════╗
+                                ║███░███░░░░██░░░░░║
+                                ║░██░████░░░██░░░░░║
+                                ║░░█░██░██░░██░█░░░║
+                                ║░░░░██░░██░██░██░░║
+                                ║░░░░██░░░████░███░║
+                                ╚══════════════════╝[/color]
+                                            [head=3]NanoTrasen[/ head]
+                                     [head=3][color=#9fed58]Funeral Intructions[/color][/head]
+[bold]─────────────────────────────────────────[/bold]
+
+[head=2][italic]Upon untimely demise, ___[color=#5b97bc]NAME HERE[/color]___ requests the following:[/italic][/head] 
```

Comment:
you could add a newline after the fill-in spot so that "requests the following" is always on the line below, and the use that line to add the ‾ characters so that the fill-in spot actually has an unbroken line under it